src/components/popovers/Popover.jsx

```diff
@@ -24,6 +24,38 @@ export const getPopperPlacement = (position, alignment) => {
   return `${position}${variation}`;
 };
 
+let overflowStyle = null;
+
+const useDisableScroll = (disableScrollRef, isDisabled) => {
+  useEffect(() => {
+    if (disableScrollRef) {
+      const domNode = disableScrollRef.current;
```

Comment:
this is annoying, but there's more than one way a ref can be defined now (old way, useRef) - fall back directly to the ref reference if `current` is undefined is a way to support both

```suggestion
      const domNode = disableScrollRef.current || disableScrollRef;
```